.github/workflows/test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [12.x, 14.x]
```

Comment:
This is fallout from upgrading webpack, which was required to resolve a bug and add the demo page.